app/components/services/services.html

```diff
@@ -45,6 +45,12 @@
                 </a>
               </th>
               <th>
+                <a ui-sref="services" ng-click="order('Status')">
+                  Status
+                  <span ng-show="sortType == 'Status' && !sortReverse" class="glyphicon glyphicon-chevron-down"></span>
+                  <span ng-show="sortType == 'Status' && sortReverse" class="glyphicon glyphicon-chevron-up"></span>
```

Comment:
I agree with that, but at the moment this code is duplicated in a lot of places so it makes no sense to change it only here. I've planned to refactor how tables are managed inside Portainer, see my comment https://github.com/portainer/portainer/pull/565#issuecomment-279971963 

If we can manage to refactor all tables into a single Angular component, we will also be able to tackle these issues:

* https://github.com/portainer/portainer/issues/568
* https://github.com/portainer/portainer/issues/630